boards/b-l072z-lrwan1/Makefile.include

```diff
@@ -12,5 +12,8 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 export DEBUG_ADAPTER ?= stlink
 export STLINK_VERSION ?= 2-1
 
+# do a reset before halt when starting debugger
```

Comment:
Same here, it is not a `reset` before `halt` but a `reset halt` openocd command.
http://openocd.org/doc/html/General-Commands.html#resetcommand

Reply:
Indeed "The optional parameter specifies what should happen after the reset" so there's a reset before halt (halt a an optional parameter) :)